user_guide_src/source/concepts/factories.rst

```diff
@@ -15,6 +15,11 @@ What are Factories?
 Like :doc:`./services`, **Factories** are an extension of autoloading that helps keep your code
 concise yet optimal, without having to pass around object instances between classes.
 
+Factories are similar to CodeIgniter 3's ``$this->load`` in the following points:
```

Comment:
Is it worth adding CI3 references at this point? I don't know that the majority of our user base is coming from there, or recently enough to remember the CI3 mechanics.

Reply:
I see no harm. It can help some people to understand basic concepts.